.github/workflows/i18n.yml

```diff
@@ -1,63 +1,46 @@
-name: Locale Updater
+name: Update locale files
 
 on:
   push:
     branches:
-      - master
       - main
+      - master
+
+permissions:
+  pull-requests: write
+  contents: write
 
 jobs:
   locale-updater:
+    if: github.repository == 'jazzband/djangorestframework-simplejwt'
     name: Locale updater
     runs-on: ubuntu-latest
-    permissions:
-      pull-requests: write
```

Comment:
Why make this global? I know this workflow only has one job but I'm just curious.

Reply:
it's been 2 months since thinking about this PR, I've forgotten if there's a need to make this global. Would you like to keep is job scoped? Like you said, they are effectively the same since there is only 1 job.

Reply:
moved to job sscope